src/cm_common/cm_stat.h

```diff
@@ -326,6 +326,10 @@ extern "C"
     unsigned int bt_num_splits;
     unsigned int bt_num_merges;
     unsigned int bt_num_get_stats;
+    unsigned int bt_copy_leaf_nolatch_success_cnt;
+    unsigned int bt_copy_leaf_nolatch_fails_cnt;
+    unsigned int bt_acquire_tran_bcb_area_success_cnt;
+    unsigned int bt_acquire_tran_bcb_area_fails_cnt;
```

Comment:
I don't think the last two are necessary, since they show the number of success and failure of memory allocation. 
please also remove `heap_acquire_*`.